pyadjoint/tape.py

```diff
@@ -312,12 +312,19 @@ def enable_checkpointing(self, schedule):
         Args:
             schedule (checkpoint_schedules.schedule): A schedule provided by the
             checkpoint_schedules package.
+            gc_collect_opts (dict): A dictionary of options to be passed to the
+            garbage collector. The keys considered here is `timestep_frequency`.
+            Optionally the user can provide the `generation` key. The default value
```

Comment:
Add generation and say correctly about the key for checkpointing and the generation key.

Reply:
Add two args (gc_timestep_frequency and gc_generation) instead of adding a dictionary.

Reply:
Done.